### ISSUE TITLE
azurerm_mssql_database support for Hyperscale Serverless

### ISSUE THREAD

**Issue by dlynch33** [hashicorp/terraform-provider-azurerm#20547]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Community Note

* Please vote on this issue by adding a :thumbsup: [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version

1.3.9

### AzureRM Provider Version

3.41.0

### Affected Resource(s)/Data Source(s)

azurerm_mssql_database

### Terraform Configuration Files

```hcl
resource "azurerm_mssql_database" "db" {
  name                             = var.database_name
  server_id                        = azurerm_mssql_server.primary.id
  sku_name                         = "HS_S_Gen5_8" # Serverless Hyperscale G5 w/ 1-8 vCores
  min_capacity                     = 1
}
```

### Debug Output/Panic Output

```shell
2023-02-17T18:33:50.1157258Z Error: invalid value for sku_name (This is not a valid sku name. For example, a valid sku name is 'GP_S_Gen5_1','HS_Gen4_1','BC_Gen5_2', 'ElasticPool', 'Basic', 'S0', 'P1'.)
2023-02-17T18:33:50.1158257Z 
2023-02-17T18:33:50.1159096Z   with module.mssql_server[0].azurerm_mssql_database.db,
2023-02-17T18:33:50.1160069Z   on .terraform/modules/mssql_server/main.tf line 250, in resource "azurerm_mssql_database" "db":
2023-02-17T18:33:50.1160714Z  250:   sku_name                         = var.mssql_database_sku
2023-02-17T18:33:50.1160895Z 
2023-02-17T18:33:50.1658991Z ##[warning]Can't find loc string for key: TerraformPlanFailed
2023-02-17T18:33:50.1682610Z ##[error]Error: TerraformPlanFailed 1
```

### Expected Behaviour

Should create MS SQL Database, Hyperscale tier, Serverless with vCore minimum and maximum of 1 and 8.

### Actual Behaviour

See error above.

**Comment by mikemadeja** [comment]
Hello, I've created a PR for this that was merged. It should be in the 3.47.0 release. 

Thanks!

Mike

**Comment by dlynch33** [comment]
Excellent!  Thanks Mike, much appreciated!

**Comment by manicminer** [comment]
Closed in #20875
